public_html/wp-content/plugins/notify-central-on-pending-posts.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Plugin Name: Notify WordCamp Central on pending posts
+ * Plugin URI: http://wordcamp.org
+ * Description: Send email notification to WordCamp Central when post status becomes pending.
+ * Version: 1.0
+ *
+ * Heavily inspired from Pending Submission Notifications plugin by Razvan Horeanga.
+ */
+
+namespace Notify_Central_Pending_Posts;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Invalid request.' );
+}
+
+add_action( 'transition_post_status', __NAMESPACE__ . '\send_notification_email', 10, 3 );
+
+/**
+ * Send the notification email.
+ *
+ * @param string  $new_status New post status.
+ * @param string  $old_status Old post status.
+ * @param WP_Post $post       Post object.
+ */
+function send_notification_email( $new_status, $old_status, $post ) {
+	if ( 'pending' === $new_status && user_can( $post->post_author, 'edit_posts' ) ) {
```

Comment:
not sure of the purpose of the `edit_posts` check here - would that not be covered by edit privileges anyway?